### DISCUSSION TITLE
Website is down for several days

### DISCUSSION THREAD

**Discussion by xoex** [JedWatson/react-select#5600] [General]
Hi guys.
React-select.com is down for several days, 
It's hard to use library without documentation

**Comment by xoex** [comment]
@JedWatson

**Reply by manjushsh** [reply]
@xoex, No it isn't. 
![image](https://user-images.githubusercontent.com/94426452/229263626-fb6cc082-0f2f-40e6-b939-34482e51eb98.png)
https://react-select.com/home

**Comment by Methuselah96** [comment]
It seems to be working for me. Maybe something particular about where you're connecting from?
